src/ServiceControl.AcceptanceTests/Recoverability/MessageFailures/When_processing_message_with_missing_metadata_failed.cs

```diff
@@ -89,30 +98,23 @@ protected override TransportOperations CreateMessage(MyContext context)
                     var headers = new Dictionary<string, string>
                     {
                         [Headers.MessageId] = context.MessageId,
-                        [Headers.ProcessingEndpoint] = context.EndpointNameOfReceivingEndpoint,
-                        ["NServiceBus.ExceptionInfo.ExceptionType"] = "2014-11-11 02:26:57:767462 Z",
-                        ["NServiceBus.ExceptionInfo.Message"] = "An error occurred while attempting to extract logical messages from transport message NServiceBus.TransportMessage",
-                        ["NServiceBus.ExceptionInfo.InnerExceptionType"] = "System.Exception",
-                        ["NServiceBus.ExceptionInfo.Source"] = "NServiceBus.Core",
-                        ["NServiceBus.ExceptionInfo.StackTrace"] = string.Empty,
                         ["NServiceBus.FailedQ"] = Conventions.EndpointNamingConvention(typeof(Failing)),
-                        ["NServiceBus.TimeOfFailure"] = "2014-11-11 02:26:58:000462 Z"
+                        [Headers.ProcessingMachine] = "unknown", // This is needed for endpoint detection to work since "host" is required, endpoint name is detected from the FailedQ header
                     };
+
                     if (context.TimeSent.HasValue)
                     {
                         headers["NServiceBus.TimeSent"] = DateTimeOffsetHelper.ToWireFormattedString(context.TimeSent.Value);
                     }
 
-                    var outgoingMessage = new OutgoingMessage(context.MessageId, headers, new byte[0]);
+                    var outgoingMessage = new OutgoingMessage(context.MessageId, headers, Array.Empty<byte>());
 
-                    return new TransportOperations(
-                        new TransportOperation(outgoingMessage, new UnicastAddressTag("error"))
-                    );
+                    return new TransportOperations(new TransportOperation(outgoingMessage, new UnicastAddressTag("error")));
                 }
             }
         }
 
-        public class MyContext : ScenarioContext
```

Comment:
I see several changes in this diff which are unrelated to the actual change. IMHO it woul be better to do these in a separate PR.

Reply:
I've rolled back the other cleanup, fixes to this file feels close enough to be included